ember-bootstrap/addon/components/bs-button.hbs

```diff
@@ -1,8 +1,14 @@
-{{! @glint-nocheck }}
 <button
   disabled={{this.__disabled}}
   type={{if @attrTypePrivateWorkaround @attrTypePrivateWorkaround "button"}}
-  class="btn {{if @active "active"}} {{if (macroCondition (macroGetOwnConfig "isNotBS5")) (if this.block "btn-block")}} {{bs-size-class "btn" @size}} {{bs-type-class "btn" @type default="secondary" outline=@outline}}"
+  class="btn
```

Comment:
While this does work, I think the newlines will be included while rendering.
Is this something we want to look out for?

Reply:
I just fixed that!

I think our prettier setup is not right, wanted to come up with a PR to fix this. With pnpm and no hoisting, we don't have prettier in the root node_modules, but VS Code expects it there to work with the prettier plugin. I believe as this was not working ootb, it would reformat the code, but not based on prettier. Had to run `lint:fix` sometimes, which is unexpected, as usually prettier-in-the-editor should be in-sync with prettier-in-lint-scripts.

Reply:
hmm, should the prettier configuration be in the root project?

That aside, I did like the indentation as it was before, it's just a shame that it generates all those newlines. Maybe we can add a helper in the future that strips all the unneccesary whitespaces from such class lists.

Reply:
Having a build plugin which strips whitespaces would be great. But I'm not sure if something like that already exists in Ember ecosystem.

Reply:
There is this: https://github.com/mainmatter/ember-hbs-minifier 

But not sure if it's able to only run on our addon. Also when migrating to v2, we would want to have something that runs ahead of time as part of our rollup build...

Reply:
Yeah. I have more thinking in a v2 addon world. I feel it's too risky and unnecessary churn introducing such tooling before v2 conversion.